controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -2,7 +2,11 @@ controls:
 - id: '5'
   title: Policies
   status: pending
-  rules: []
+  rules:  
+    ### Helper Rules
+    ### This is a helper rule to fetch the required api resource for detecting OCP version
+    - version_detect_in_ocp
+    - version_detect_in_hypershift
```

Comment:
@Vincent056 What do you think of keeping the rules in the profile?
Like @rhmdnd did in https://github.com/ComplianceAsCode/content/pull/10833/files#diff-fbb87f91441059e74851db5faf811abdba39811d1b39209bc72baacb45a19c81R32 ?

Reply:
Just FTR, these rules do not pertain to any control, but are kindof internal CO rules. (Not saying we can't move them to a control file, just saying that they are control-less)

Reply:
Exactly, as they are control-less it makes more sense to me keep them in the `.profile` file.